WordPress/src/main/java/org/wordpress/android/ui/mediapicker/MediaItem.kt

```diff
@@ -14,21 +19,91 @@ data class MediaItem(
     val mimeType: String? = null,
     val dataModified: Long
 ) {
-    sealed class Identifier : Parcelable {
-        @Parcelize
-        data class LocalUri(val value: @RawValue UriWrapper) : Identifier()
+    enum class IdentifierType {
+        LocalUri,
```

Comment:
this is very minor but I'd rather have these looking like `LOCAL_URI` and so on. The reason is that things like `data class LocalUri(val value: UriWrapper) : Identifier(LocalUri)` are a bit confusing - to see which `LocalUri` you're using. WDYT?